### PR TITLE
Initial commit: Assignment 2

### PR DESCRIPTION
**Pull request by urvashichaudhary13**
Assignment - 2 
Question 1 - Write a program to give a random output between 1 to 10 (AC: 2 simultaneous answers could not be same)
Question 2 - Write a program to take input(age, name, phone no.) from user and display it
Question 3 - Write a function expression that takes in another function as an argument
Question 4 - Write a function expression that takes in a number and returns its square.
Question 5 - Write a function expression that takes in two numbers and returns their sum.
Question 6- Write a function expression that takes in a number and returns true if it's even and false if it's odd.
Question 7 - All of the above questions from 3-6 needs to be done with arrow functions also

### CODE REVIEW

File: assignment1/problem1.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+Question 1 - Write a program to give a random output between 1 to 10 (AC: 2 simultaneous answers could not be same)
+*/
+
+let previousOutput = null
+let min = 1;
```
Comment (choudharyajay) [inline assignment1/problem1.js:6]: Please use const instead of let as you are not re-assigning any values here.

File: assignment1/problem6.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/* 
+Question 6- Write a function expression that takes in a number and returns true if it's even and false if it's odd.
+*/
+
```
Comment (choudharyajay) [inline assignment1/problem6.js:4]: There is no need to use if condition here - as we can simple return the comparison like this -

const evenOrOdd = (num) => {
          return num === 0 || num % 2 === 0;
   }

**Review by choudharyajay — CHANGES_REQUESTED** [review]

File: assignment1/problem1.js (hunk 1 above)
Comment (urvashichaudhary13) [reply assignment1/problem1.js:6]: done

File: assignment1/problem6.js (hunk 2 above)
Comment (urvashichaudhary13) [reply assignment1/problem6.js:4]: sure

**Review by choudharyajay — APPROVED** [review]

**Pull request merged**